gpt_computer_agent/__init__.py

```
@@ -1,7 +1,288 @@
+import subprocess
+import requests
+
+
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+
+
+class instance:
+    def __init__(self, url, tasks=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)